megamek/src/megamek/common/MoveStep.java

```diff
@@ -3962,6 +3964,10 @@ public boolean useSpheroidAtmosphere(IGame game, Entity en) {
      * @return An {@link ArrayList} of {@link Coords} containing buildings within a dropship's landing zone.
      */
     public ArrayList<Coords> getCrushedBuildingLocs() {
```

Comment:
Does the caller add to it? Or could the caller just receive say Collections.emptyList instead?

Reply:
No, but it looks like I introduced a few bugs with that optimization.

Reply:
Found a place where this collection was being added to without being initalized.